src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1346,6 +1297,89 @@ private void UpdateSymmetryIcon()
         }
     }
 
+    /// <summary>
+    ///   Lock / unlock a single organelle that need a nucleus
+    /// </summary>
+    private void UpdatePartAvailability(bool hasNucleus, OrganelleDefinition organelle)
+    {
+        // Item could be a non-implemented organelle so check if it really exists first.
+        if (!placeablePartSelectionElements.ContainsKey(organelle))
```

Comment:
From what I can tell this condition is never true now as UpdatePartAvailability is called for only things in the dictionary, so the organelle always exists.